pkgs/development/python-modules/acstore/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "acstore";
+
+  version = "20230226";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-MDeWOU1cv6dW8LL2UmdUbSwm0vg4YfxvpTDTb15+/+s=";
+  };
+
+  meta = with lib; {
+    description = "ACStore, or Attribute Container Storage, provides a stand-alone implementation to read and write attribute container storage files";
+    downloadPage = "https://github.com/log2timeline/acstore/releases";
+    homepage = "https://github.com/log2timeline/acstore";
```

Not much content in the changelog at the moment, but perhaps there will be more in the future.

```suggestion
    homepage = "https://github.com/log2timeline/acstore";
    changelog = "https://github.com/log2timeline/acstore/releases/tag/${version}";
```